classes/components/forms/context/PKPEmailSetupForm.inc.php

```diff
@@ -39,6 +41,22 @@ public function __construct($action, $locales, $context) {
 
 		$canEnvelopeSender = \Config::getVar('email', 'allow_envelope_sender');
 
+		$this->addField(new FieldRadioInput('copySubmissionAckPrimaryContact', [
+			'label' => __('manager.setup.notifications.copySubmissionAckPrimaryContact'),
+			'isMultilingual' => false,
+			'options' => [
+				['value' => true, 'label' => __('manager.setup.notifications.copySubmissionAckPrimaryContact.enabled')],
+				['value' => false, 'label' => __('manager.setup.notifications.copySubmissionAckPrimaryContact.disabled')],
+			],
+			'value' => $context->getData('copySubmissionAckPrimaryContact'),
+		]));
+
+		$this->addField(new FieldItems('copySubmissionAckAddress', [
+			'label' => __('manager.setup.notifications.copySubmissionAckAddress'),
+			'tooltip' => __('manager.setup.notifications.copySubmissionAckAddress.help'),
+			'value' => $context->getData('copySubmissionAckAddress'),
+		]));
```

Comment:
Use the following `label`:

> Notify Anyone

And the following `description`:

> A copy of the submission acknowledgement email will be sent to any of the email addresses entered here. Separate multiple email addresses with a comma. Example: dbarnes@example.com,minoue@example.com

Then get rid of the `tooltip`.